AtReconstruction/AtPatternRecognition/AtTrackFinderTC.cxx

```diff
@@ -25,6 +27,11 @@ constexpr auto cRED = "\033[1;31m";
 constexpr auto cYELLOW = "\033[1;33m";
 constexpr auto cNORMAL = "\033[0m";
 constexpr auto cGREEN = "\033[1;32m";
+std::vector<AtTrack *> candTrackPool;
+std::vector<AtTrack> mergedTrackPool;
+Bool_t fEnableSingleVertexTrack = kTRUE;
+Double_t fClusterSize = 20.0;
+
```

Comment:
These should be members of the class. If they're changed they require rebuilding - they should be changeable on a per-instance bases.

Technically the pools can live here, but right now they are in the global namespace which is bad. If they're defined in the file like this, they should live in an anonymous namespace like

```cpp
namespace {
std::vector<AtTrack *> candTrackPool;
std::vector<AtTrack> mergedTrackPool;
}
```
but I don't really like that solution since it breaks with the patterns used in the rest of the codebase